requirements.txt

```diff
@@ -1,4 +1,5 @@
 click
+jsonlines
```

Comment:
What additional does this package provide that `my_file.write(json.dumps(obj))` doesn't?

Reply:
None - removing it

Reply:
Is `'\n'` the right character for returns in .jsonlines files?